phoenix-core/src/main/java/org/apache/phoenix/query/QueryServicesOptions.java

```diff
@@ -284,14 +288,18 @@
     public static final int DEFAULT_COLUMN_ENCODED_BYTES = QualifierEncodingScheme.TWO_BYTE_QUALIFIERS.getSerializedMetadataValue();
     public static final String DEFAULT_IMMUTABLE_STORAGE_SCHEME = ImmutableStorageScheme.SINGLE_CELL_ARRAY_WITH_OFFSETS.toString();
     public static final String DEFAULT_MULTITENANT_IMMUTABLE_STORAGE_SCHEME = ImmutableStorageScheme.ONE_CELL_PER_COLUMN.toString();
+    public  final static String DEFAULT_PHOENIX_QUERY_SERVER_CLUSTER_BASE_PATH = "/phoenix";
+    public  final static String DEFAULT_PHOENIX_QUERY_SERVER_SERVICE_NAME = "queryserver";
+    public  final static String DEFAULT_PHOENIX_QUERY_SERVER_ZK_ACL_USERNAME = "phoenixuser";
+    public  final static String DEFAULT_PHOENIX_QUERY_SERVER_ZK_ACL_PASSWORD = "Xsjdhxsd";
```

Comment:
> I replied saying that we can close this PR and open another ticket, where I will fix the SASL ACLs.

Implementing SASL acls is one thing -- my above comment was about not forgetting to document the lack of initial support (much less the feature itself).

We should make sure that a documentation issue gets filed, not that you build something that rots because no one knows how to use it!